packages/combo-box/test/toggling-dropdown.test.js

```diff
@@ -356,7 +359,9 @@ describe('toggling dropdown', () => {
     });
   });
 
-  (isIOS ? describe : describe.skip)('external focus (initially)', () => {
+  // FIXME: isTouch check returns true for WebKit (Playwright).
+  // See https://github.com/vaadin/web-components/issues/257
+  (isTouch ? describe : describe.skip)('external focus (initially)', () => {
```

Comment:
Previously, the test wasn't run at all. The change makes it run with WebKit to cover this block:

https://github.com/vaadin/web-components/blob/b4fa5d42bf3b35c36c4c075e79d2a3b2dde4d7b1/packages/combo-box/src/vaadin-combo-box-mixin.js#L852-L856